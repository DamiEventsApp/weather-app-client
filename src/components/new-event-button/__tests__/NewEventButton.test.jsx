import React from 'react';
import { shallow } from 'enzyme';
import NewEventButton from '../NewEventButton.component.jsx';
import '../../../../setupTests.js';

describe('NewEventButton', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<NewEventButton />);
        expect(wrapper).toMatchSnapshot();
        // On the first run of this test, Jest will generate a snapshot file automatically.
    });
});