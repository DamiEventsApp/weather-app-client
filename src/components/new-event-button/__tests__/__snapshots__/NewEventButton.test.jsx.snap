// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewEventButton renders correctly 1`] = `ShallowWrapper {}`;
